docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,559 @@
+
+For the simplicity of this guide, certain best practices will be provided with links to additional documents, enabling you to explore the topic in more detail.
+
+:::warning
+Following this guide will incur costs on your Cloud provider account, namely for the managed Kubernetes service, running Kubernetes nodes in EC2, Elastic Block Storage (EBS), traffic between regions, and S3. More information can be found on [AWS](https://aws.amazon.com/eks/pricing/) and their [pricing calculator](https://calculator.aws/#/) as the total cost varies per region.
+:::
+
+:::warning
+Please have a look at our [dual-region concept documentation](#) <!-- TOOD: link reference --> before continuing to understand the current limitations and restrictions of this setup, as well as the disclaimer concerning support from Camunda.
+:::
+
+## Outcome
+
+Following this tutorial and steps will result in:
+
+- Two Amazon EKS Kubernetes clusters running the latest Kubernetes version with each four nodes ready for Camunda 8 installation.
+- The [EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) is installed and configured, which is used by the Camunda 8 Helm chart to create [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+- A [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) between the two EKS clusters, allowing cross-cluster communication between different regions.
+- An [Amazon Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) bucket for [ElasticSearch backups](https://www.elastic.co/guide/en/elasticsearch/reference/current/repository-s3.html)
+
+## Environment prerequisites
+
+To streamline the execution of the subsequent commands, it is recommended to export multiple environment variables.
+
+The following are the required environment variables with some example values:
+
+```bash
+# The AWS regions of your cluster 1 and 2
+export REGION_1=eu-west-2
+export REGION_2=eu-west-3
+
+# The cluster_name of your cluster 1 and 2
+# default based on the tutorial is the following
+export CLUSTER_1=cluster-london
+export CLUSTER_2=cluster-paris
+
+# The namespaces for each region where Camunda 8 should be running in
+export CAMUNDA_NAMESPACE_1=camunda-london
+export CAMUNDA_NAMESPACE_2=camunda-paris
+```
+
+## Installing Amazon EKS clusters with Terraform
+
+<Tabs queryString="tf-installation">
+<TabItem value="terraform-prerequsites" label="Terraform prerequsites" default>
+
+1. Create an empty folder to place your Terraform files in.
+2. Create a `config.tf` with the following setup:
+
+<details>
+  <summary>config.tf</summary>
+  <summary>
+
+```hcl reference title="config.tf"
+https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/terraform/config.tf#L5-L51
+```
+
+  </summary>
```

Comment:
will keep it open for now, not sure whether still needed with the current suggestions.
Never noticed since I always open links with the scroll wheel press.

Reply:
Agree. Can be a bug of the Docusaurus plugging where they did not think of the corner case of when a code block is inside a collapsing block.